src/pdn/harvesters.py

```diff
@@ -81,6 +98,22 @@ def from_django_record(cls, harvester: "Harvester"):
                 "harvest_news", True),
             harvest_projects=harvester.harvester_type_specific_configuration.get(
                 "harvest_projects", True),
+            document_publication_day_filter=harvester.harvester_type_specific_configuration.get(
+                "document_publication_day_filter", None),
+            document_publication_month_filter=harvester.harvester_type_specific_configuration.get(
+                "document_publication_month_filter", None),
+            document_publication_year_filter=harvester.harvester_type_specific_configuration.get(
+                "document_publication_year_filter", None),
+            alerts_start_date_filter=harvester.harvester_type_specific_configuration.get(
+                "alerts_start_date_filter", None),
+            alerts_end_date_filter=harvester.harvester_type_specific_configuration.get(
+                "alerts_end_date_filter", None),
+            news_start_date_filter=harvester.harvester_type_specific_configuration.get(
+                "news_start_date_filter", None),
+            news_end_date_filter=harvester.harvester_type_specific_configuration.get(
+                "news_end_date_filter", None),
+            project_active_filter=harvester.harvester_type_specific_configuration.get(
+                "project_active_filter", None),
```

Comment:
`dict.get()` already returns `None` if no default is supplied, so please simplify this passage

```suggestion
            document_publication_day_filter=harvester.harvester_type_specific_configuration.get("document_publication_day_filter"),
            document_publication_month_filter=harvester.harvester_type_specific_configuration.get("document_publication_month_filter"),
            document_publication_year_filter=harvester.harvester_type_specific_configuration.get("document_publication_year_filter"),
            alerts_start_date_filter=harvester.harvester_type_specific_configuration.get("alerts_start_date_filter"),
            alerts_end_date_filter=harvester.harvester_type_specific_configuration.get("alerts_end_date_filter"),
            news_start_date_filter=harvester.harvester_type_specific_configuration.get("news_start_date_filter"),
            news_end_date_filter=harvester.harvester_type_specific_configuration.get("news_end_date_filter"),
            project_active_filter=harvester.harvester_type_specific_configuration.get("project_active_filter"),
```

Reply:
updated @ricardogsilva